src/pages/_images/oms2.mmd

```diff
@@ -0,0 +1,10 @@
+flowchart LR
+    subgraph Box1[Adobe Commerce]
+        A1[New Order] --> A2[I/O Events to Adobe Commerce]
+        A3[AC Extenstion that adds custom data to GQL or custom calcultations] --> A4[GQL]
```

Comment:
GH isn't letting me make a suggestion here.

Change line 4 to 

A3[Extension that adds custom data to GraphQL or performs custom calculations] --> A4[GraphQL]